README.md

```diff
@@ -1951,43 +1951,52 @@ hazelcastClient.getFlakeIdGenerator('myFlakeIdGenerator').then(function (gen) {
 
 This chapter explains when various events are fired and describes how you can add event listeners on a Hazelcast Node.js client. These events can be categorized as cluster and distributed data structure events.
 
-### 7.5.1. Cluster Events
+### 7.5.1. Listening for Cluster Events
 
 You can add event listeners to a Hazelcast Node.js client. You can configure the following listeners to listen to the events on the client side:
 
 * Membership Listener: Notifies when a member joins to/leaves the cluster, or when an attribute is changed in a member.
 * Distributed Object Listener: Notifies when a distributed object is created or destroyed throughout the cluster.
 * Lifecycle Listener: Notifies when the client is starting, started, shutting down and shutdown.
 
-#### 7.5.1.1. Listening for Member Events
+#### 7.5.1.1. Membership Listener
 
-You can add the following types of member events to the `ClusterService`.
+The Membership Listener interface has functions that are invoked for the following events.
 
 * `memberAdded`: A new member is added to the cluster.
 * `memberRemoved`: An existing member leaves the cluster.
 * `memberAttributeChanged`: An attribute of a member is changed. See the [Defining Member Attributes section](https://docs.hazelcast.org/docs/latest/manual/html-single/index.html#defining-member-attributes) in the Hazelcast IMDG Reference Manual to learn about member attributes.
 
-The `ClusterService` object exposes an `ClusterService.on()` function that allows one or more functions to be attached to the member events emitted by the object.
+For `memberAdded` and `memberRemoved` events, a `MembershipEvent` object is passed to the listener function.
 
-The following is a membership listener registration by using the `ClusterService.on()` function.
+After you create your listener object, you can configure your cluster to include the membership listener. You can also add one or more membership listener.
```

Comment:
After you create the listener object, you can configure your cluster to include the membership listener. You can also add one or more membership listeners.